### PR TITLE
[NP-7395] Add CrunchLab side view

### PR DESCRIPTION
**Pull request by KernelDeimos**
PR so clicking a capability in CrunchLab does this:
image

main reason for doing this is so I can later add options to test a capability in different wizard configurations

### CODE REVIEW

File: src/foam/u2/borders/SideViewBorder.js [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+      height: 100%;
+      top: 0;
+      right: 0;
+      padding: 10px;
+      border-left: 4px solid rgba(0,0,0,0.4);
+      display: flex;
+      flex-direction: column;
+      gap: 10px;
+    }
+    ^side^open {
```
Comment (kgrgreer) [inline src/foam/u2/borders/SideViewBorder.js:33]: Should there be a space between these?

File: src/foam/u2/borders/SideViewBorder.js [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+      width: 50%;
+      background-color: rgba(255,255,255,0.7);
+      visibility: visible;
+    }
+    ^side ^container {
+      flex-grow: 1;
+      overflow-x: hidden;
+      overflow-y: scroll;
```
Comment (sarthak-marwaha) [inline src/foam/u2/borders/SideViewBorder.js]:
```suggestion
      overflow-y: auto;
```

File: src/foam/u2/borders/SideViewBorder.js (hunk 1 above)
Comment (KernelDeimos) [reply src/foam/u2/borders/SideViewBorder.js:33]: Makes sense to me, easier cognitive processing that way - fixed

Comment (sarthak-marwaha) [reply src/foam/u2/borders/SideViewBorder.js:33]: that wont work `.class1.class2` selects all elements with class 1 and class 2 but `.class1 .class2` selects all class2 that are descendants of class 1

Comment (KernelDeimos) [reply src/foam/u2/borders/SideViewBorder.js:33]: Oh I thought kgr meant a line between the two blocks. Yeah it's still `^side^open`

**Pull request merged**